src/ripple/basics/Slice.h

```diff
@@ -73,12 +74,20 @@ class Slice
 
         This may be zero for an empty range.
     */
+    /** @{ */
     std::size_t
     size() const noexcept
     {
         return size_;
     }
 
+    std::size_t
+    length() const noexcept
+    {
+        return size_;
+    }
```

Comment:
Consider making this `constexpr`.

Reply:
Not a big deal, but string_view has `size` too.  `length` is for legacy support.